official/resnet/resnet_run_loop.py

```diff
@@ -384,16 +405,37 @@ def input_fn_eval():
       benchmark_logger = logger.BenchmarkLogger(flags.benchmark_log_dir)
       benchmark_logger.log_estimator_evaluation_result(eval_results)
 
+  # Return the classifier after all epochs are completed.
+  return classifier
+
+
+def export_savedmodel(classifier, export_dir, shape, batch_size):
```

Comment:
I would suggest move this to some util file since it has nothing to do with the model or any internal variable for resnet. I would expect a lot of other model will have function like this too if they want export the model.

Reply:
+1

Reply:
I went with @robieta 's suggestion above and included the steps in renset_main, which removes the need for this function.